FDTD/FDTD.f90

```diff
@@ -449,9 +503,17 @@ subroutine dt_evolve_Ac_2dc()
         & -J_m(:,ix_m,iy_m) * 4.0*pi*(dt**2) - RR(:)*(c_light*dt)**2 )
     end do
   end do
+!$omp end parallel do
+
   ! Boundary condition
-  Ac_new_m(:,:,NYvacB_m-1)=Ac_new_m(:,:,NYvacB_m)
-  Ac_new_m(:,:,NYvacT_m+1)=0.0d0
+!$omp parallel do default(none) &
+!$    private(ix_m) &
+!$    shared(Ac_new_m,NXvacL_m,NXvacR_m,NYvacB_m,NYvacT_m)
+  do ix_m=NXvacL_m,NXvacR_m
```

Comment:
The range of this `do` loop should be extended as `do ix_m=NXvacL_m-1,NXvacR_m+1`.